distribution/kernel/carbon-home/dbscripts/mysql_cluster.sql

```diff
@@ -393,6 +393,16 @@ CREATE TABLE UM_HYBRID_USER_ROLE(
             PRIMARY KEY (UM_ID, UM_TENANT_ID)
 )ENGINE NDB;
 
+CREATE TABLE UM_HYBRID_GROUP_ROLE(
+            UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+            UM_GROUP_NAME VARCHAR(255),
+            UM_ROLE_ID INTEGER NOT NULL,
+            UM_TENANT_ID INTEGER DEFAULT 0,
+            UM_DOMAIN_ID INTEGER,
+            UNIQUE (UM_GROUP_NAME, UM_ROLE_ID, UM_TENANT_ID, UM_DOMAIN_ID),
```

Comment:
Any specific reason why we don't have foreign key references in this table?

Reply:
Added foreign keys for both UM_HYBRID_USER_ROLE and UM_HYBRID_GROUP_ROLE.